.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Install Rust
       uses: hecrj/setup-rust-action@v1
       with:
-        rust-version: 1.65
+        rust-version: 1.69
```

Comment:
If this crate needs a Rust compiler 1.69 or later, the version in Cargo.toml also needs to be updated.